src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/MergeBuildManifestsTests.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+
+namespace Microsoft.DotNet.Maestro.Tasks.Tests
+{
+    [TestFixture]
+    public class MergeBuildManifestsTests
+    {
+        // TODO: Find whatever is making this not work concurrently.
```

Comment:
You still doing this? :)

Reply:
Yep, just needed to see it fresh today, spent too long staring at it yesterday and ran out of ideas.